platform/android/DEVELOPING.md

```diff
@@ -33,101 +13,64 @@ git submodule update --init --recursive
 cd platform/android
 ```
 
-## Installing dependencies
+## Requirements
 
-These dependencies are required for all operating systems and all platform targets.
+[Android Studio](https://developer.android.com/studio) needs to be installed.
 
-- Latest stable [Android Studio](https://developer.android.com/studio/index.html)
-- Update the Mapbox Maps SDK for Android with the latest
-  - Android SDK Build-Tools
-  - Android Platform-Tools
-  - Android SDK Tools
-  - CMake
-  - NDK
-  - LLDB
-- Modern C++ compiler that supports `-std=c++14`\*
-  - clang++ 3.5 or later or
-  - g++-4.9 or later
-- [Node.js](https://nodejs.org/)
-  - make sure [npm](https://www.npmjs.com) is installed as well
-- [ccache](https://ccache.samba.org/) (optional)
+Open the `platform/android` directory to get started.
 
-**Note**: We partially support C++14 because GCC 4.9 does not fully implement the
-final draft of the C++14 standard. More information in [DEVELOPING.md](../../DEVELOPING.md).
+## Setting an API Key
 
-**Note**: On macOS you can install clang with installing the [Apple command line developer tools](https://developer.apple.com/download/).
-
-## Opening the project
+_The test application (used for development purposes) uses MapTiler vector tiles, which require a MapTiler account and API key._
```

Comment:
Could we have the test application use the MapLibre demotiles instead, so the developer wouldn't need to create a MapTiler account?

https://github.com/maplibre/demotiles/blob/gh-pages/README.md

Reply:
It uses the demotiles by default, but many features cannot be demoed with the demotiles style...